test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2154,7 +2155,7 @@
           (is (ca/crl-expires-in-n-days? path-as-string settings 6))))))
 
 (deftest overwrite-existing-crl!-test
-  (let [settings (testutils/ca-sandbox! cadir)
+  (let [settings (testutils/ca-sandbox! bundle-cadir)
```

Comment:
the `bundle-cadir` has a root CA and an intermediate, so changing this allows that path to be tested.